### ISSUE TITLE
Windscreen Washer Fluid Level Warning

### ISSUE THREAD

**Issue by Kaputt4** [tillsteinbach/VWsFriend#612]
Hello, and a huge thank you to [@tillsteinbach](https://github.com/tillsteinbach) for your incredible work on this project!

I'm reaching out to report an issue I encountered with my ID.3 car running We Connect version 3.2. Recently, the windscreen washer fluid level warning has appeared several times. The warning is categorized as `Others` within the Volkswagen mobile app.

<img src='https://github.com/tillsteinbach/VWsFriend/assets/73181608/ed71c4f6-89f8-4f3b-82cf-d031222f1d3c' width='250'>

I found the following data in the `warning_lights` database:

```bash
vwsfriend=# select text, category, "messageId", "serviceLead", "customerRelevance", priority from warning_lights order by id desc limit 2;
      text      | category | messageId | serviceLead | customerRelevance | priority
----------------+----------+-----------+-------------+-------------------+----------
 TEXT_NOT_FOUND | UNKNOWN  | 0xA227    | f           |                   |      125
 TEXT_NOT_FOUND | UNKNOWN  | 0xA227    | f           |                   |      125
(2 rows)
```

Additionally, the `vwsfriend` container logs displayed this warning:

```bash
2024-03-06T09:01:45+0000:WARNING:addressable:/vehicles/<VIN>/domains/vehicleHealthWarnings/warningLights/warningLights/0xA227/category: An unsupported category: OTHER was provided, known values are [LIGHTING, TIRE, ENGINE, unknown category] please report this as a bug
```

As Grafana dashboards use the `text` filed, `New warning: TEXT_NOT_FOUND` appears.

If there's anything I can assist with to help resolve this problem, please don't hesitate to reach out. Once again, thank you for your hard work and dedication to this project!

**Comment by MattiJarvinen** [comment]
Can confirm.

sudo docker logs vwsfriend-vwsfriend-1 2>&1 | grep warningLights
2024-05-16T18:51:16+0000:ERROR:warning_lights_status:/vehicles/XXXX/domains/vehicleHealthWarnings/warningLights/warningLights/0xA227: warning light icon is not a base64 encoded png
2024-05-16T18:51:16+0000:WARNING:addressable:/vehicles/XXXX/domains/vehicleHealthWarnings/warningLights/warningLights/0xA227/iconColor: An unsupported iconColor: ICON_NOT_FOUND was provided, known values are [Yellow, Red, unknown color] please report this as a bug

**Comment by tillsteinbach** [comment]
Should be fixed in 0.24.5
